scripts/calculateRevenue/protocols/utils/networks.ts

```diff
@@ -0,0 +1,49 @@
+import { QueryResponse, TransactionField } from '@envio-dev/hypersync-client'
+import { getHyperSyncClient } from '../../../utils'
+import { NetworkId } from '../../../types'
+
+export async function fetchTotalGasUsed({
```

Comment:
same naming nit here: this returns total transaction fees, right?  not gas

Reply:
Changed in e36298d18e323ff1455de7bbab21b05794ed1029.